### DISCUSSION TITLE
Is it possible to start with the regular neovim startup screen?

### DISCUSSION THREAD

**Discussion by abeldekat** [LazyVim/LazyVim#364] [Q&A] [answered]
Thank you for this great project! I admire the concept.

I was hoping that setting enabled to false on goolord/alpha-nvim would bring the regular neovim startup screen back. However, the screen is blank.
Maybe some plugin activity overrides the neovim startup screen? I could not find the cause in the source code of LazyVim. The problem does not seem related to a specific plugin.

**Comment by folke** [comment, marked as answer]
Some plugins can interfere with the start screen indeed. A simple `:redraw!` clears the start screen. With plugins like noice etc, it's just not possible to keep the default start screen

**Reply by abeldekat** [reply]
Thank you for answering!

**Comment by abeldekat** [comment]
Hello @Folke,

Seven months ago I started using `LazyVim` and asked this question.
I would like to thank you for creating `LazyVim` and `lazy.nvim`. Especially, I think that the concept of a plugin as a thin container for other plugins is a novelty and a very sound architectural decision.

That said:

> Some plugins can interfere with the start screen indeed. A simple :redraw! clears the start screen. With plugins like noice etc, it's just not possible to keep the default start screen

I like to tweak...) 
The snippet in this [comment](https://github.com/folke/lazy.nvim/issues/1038#issuecomment-1733035996)  shows a working example using stock `LazyVim`.

Best regards!
